### ISSUE TITLE
Summary of running issues

### ISSUE THREAD

**Issue by Ekleog** [git-wotr/spec#16]
GitHub doesn't allow issues to be marked as blocked by another issue, so let's recap where we are here. I'll try to update this post as remaining questions are solved.

 * #2 *reproducible builds*: Needs feedback
 * #3 *commit-id, tree-id, patch-id*: In progress
 * #4 *what should be in signatures*: Needs other people commenting
 * #5 *file format for the configuration*: Blocked on #9 
 * #6 *should there be versioning*: Blocked on #4 
 * #9 *how to exchange specifications for commit checking*: Needs other people commenting
 * #10 *ignore unknown metadata fields*: Blocked on #4 
 * #12 *add a “vendor” key*: Blocked on #4 / #10 
 * #13 *git-notes or custom format*: Blocked on #3 
 * #14 *what commit-diff exactly signs*: Needs someone to actually write the text for the idea we agreed about
 * #15 *ignoring lines that don't parse as reviews*: Needs someone to actually write the text for the idea we agreed about
